src/app.controller.ts

```
@@ -10,4 +10,11 @@ export class AppController {
     console.log("héhé");
     return this.appService.getHello();
   }
+
+  @Get("test")
+  getTest(): string {
+    const date = new Date();
+    return date.getDay();
+    return "test";
+  }
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)